### PR TITLE
Fix info log source file display length

### PR DESCRIPTION
**Pull request by yiwu-arbug**
Summary:
Source code path in info log is not truncated to the correct length. Fixing it. 

Test Plan:
Build and run db_bench. Before:
```
2019/09/18-21:32:34.631181 7fdd42df6700 [_impl/db_impl_write.cc:1654] [default] New memtable created with log file: #9. Immutable memtables: 0.
```
After:
```
2019/09/18-21:36:09.226532 7f141b5f6700 [/db_impl/db_impl_write.cc:1654] [default] New memtable created with log file: #9. Immutable memtables: 0.
```

### CODE REVIEW

File: logging/logging.h [hunk 1]
```diff
@@ -19,9 +19,9 @@
 
 inline const char* RocksLogShorterFileName(const char* file)
 {
-  // 15 is the length of "logging/logging.h".
+  // 18 is the length of "logging/logging.h".
   // If the name of this file changed, please change this number, too.
-  return file + (sizeof(__FILE__) > 15 ? sizeof(__FILE__) - 15 : 0);
+  return file + (sizeof(__FILE__) > 18 ? sizeof(__FILE__) - 18 : 0);
```
Comment (siying) [inline logging/logging.h]: Can you explain why 15 is not the correct length and 18 is?

Comment (siying) [reply logging/logging.h]: Oh 18 is the length of "logging/logging.h"? Fine then....

**Comment by yiwu-arbug** [comment]
Is this ready to land? Thanks.

**Comment by yiwu-arbug** [comment]
ping

**Comment by siying** [comment]
Landing failed again... Let me retry.

**Comment by siying** [comment]
Err... Our internal tool doesn't allow us to land the PR, saying it's too old. @yiwu-arbug can you help rebase?

**Comment by yiwu-arbug** [comment]
See if it will work this time?

**Comment by yiwu-arbug** [comment]
Anyone can merge this one? Thanks!

**Pull request closed without merging**